conf/messages

```diff
@@ -36,26 +19,21 @@ connectAgentServices.start.details.p2=This service will not allow you to access
 connectAgentServices.start.details.l2.1=register a client''s trust
 connectAgentServices.start.details.l2.2=view a client''s PAYE income record
 connectAgentServices.start.p3=You will need the login details for each of the agent Government Gateway accounts your business uses for your Income Tax Self Assessment and VAT clients.
-connectAgentServices.start.p4=You will need to repeat this process for each Government Gateway account your business uses.
-connectAgentServices.start.p5=On the next screen, sign in with one of these Government Gateway accounts to start linking your existing clients.
-
+connectAgentServices.start.p4=On the next screen, sign in with one of these Government Gateway accounts to start linking your existing clients.
+connectAgentServices.start.p5=You will need to repeat this process for each Government Gateway account your business uses.
 #Already linked
 alreadyMapped.p1=This Government Gateway account is already linked to your agent services account.
 alreadyMapped.p2=If you need to link a different Government Gateway account, sign in with the login details for that account.
-
 #Incorrect account
 incorrectAccount.p1=You tried to sign into the Government Gateway with your agent services account login details.
 incorrectAccount.p2=You need to sign in with a Government Gateway account that your business uses for your Income Tax Self Assessment and VAT clients.
-
 #Not enrolled
 notEnrolled.p1=You cannot link this Government Gateway account to your agent services account.
 notEnrolled.p2=You can only link Government Gateway accounts registered to your business.
 notEnrolled.p3=If you need to link another Government Gateway account, sign in with the login details for that account.
-
 #No match
 noMatch.p1=We could not find any records that match the details you entered.
```

Comment:
The no-match page that used these has been removed, so these can be removed too

Reply:
will do that